Makefile

```diff
@@ -1,29 +1,19 @@
 .PHONY: deps test build
 
-BINARY := metadata-api
-ORG_PATH := github.com/alphagov
-REPO_PATH := $(ORG_PATH)/$(BINARY)
-
 all: test build
 
-deps:
-	gom install
-
-vendor: deps
-	mkdir -p vendor
-
-test: vendor
-	gom test -v $(REPO_PATH) \
-		$(REPO_PATH)/content_api \
-		$(REPO_PATH)/need_api \
-		$(REPO_PATH)/performance_platform \
-		$(REPO_PATH)/request
+test:
+	go test -v ./ \
+		./content_api \
+		./need_api \
+		./performance_platform \
+		./request
```

Comment:
Instead of hard-coding this list of packages here you can use `go list` to generate the list, which means it'll always be up to date. eg:
```make
test:
    go test -v $$(go list ./... | grep -v '/vendor/')
```